http_server.c

```diff
@@ -7,6 +7,8 @@
 #include "http_parser.h"
 #include "http_server.h"
 
+extern mempool_t *http_buf_pool;
```

Comment:
Don't use `extern` here. Honor the header instead.

Reply:
The issue has been fixed. Thanks, professor.